src/gui/CMakeLists.txt

```diff
@@ -33,6 +33,8 @@ SET(LMMS_SRCS
 	gui/PluginBrowser.cpp
 	gui/ProjectNotes.cpp
 	gui/RowTableView.cpp
+	gui/SetupDialog.cpp
```

Comment:
Hi! I believe this was added via the merge with master. If you look further down the file there already is a 
`gui/modals/SetupDialog.cpp`. The file was moved a while back. Did you do the merge all online? Does it compile locally?

Here is from the failing builds concerning this line but there seem to be others.

```cpp
-- Configuring done
CMake Error at src/CMakeLists.txt:100 (ADD_LIBRARY):
  Cannot find source file:

    gui/SetupDialog.cpp

  Tried extensions .c .C .c++ .cc .cpp .cxx .m .M .mm .h .hh .h++ .hm .hpp
  .hxx .in .txx


-- Generating done
-- Build files have been written to: /__w/lmms/lmms/build
Error: Process completed with exit code 1.
```